Documentation/update-target-extension-data-collection.md

```diff
@@ -0,0 +1,11 @@
+# Upgrade Target extension on Data Collection UI
+
+Target extension v2.5.0 and above on Data Collection UI supports the `Raw Request` Event Type for Adobe Target extension. Click Upgrade to install the new extension version and then [publish the configuration.](https://developer.adobe.com/client-sdks/documentation/getting-started/create-a-mobile-property/#publish-the-configuration)
```

Comment:
We should clarify that this event type is needed only for raw requests. These generic APIs shouldn't be used along with regular APIs. We can do this in a separate pr later.